friendshipper/src-tauri/src/builds/router.rs

```diff
@@ -355,6 +355,7 @@ where
         let argolabels = workflow.metadata.labels.as_ref().unwrap();
         let argoannotations = workflow.metadata.annotations.as_ref().unwrap();
 
+        #[allow(clippy::manual_inspect)]

```

Comment:
Is this from a linter suggestion?

Reply:
Yeah. At first I tried fixing this by replacing the `map` whose return value we check with `inspect` as suggested; but this just caused the linter to trigger the same error on the other instances of `map` whose return value we're not using, even after I tried switching them to `inspect`